apps/lockfile-explorer/.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+// This is a workaround for https://github.com/eslint/eslint/issues/3458
+require('@rushstack/eslint-config/patch/modern-module-resolution');
+
+module.exports = {
+  extends: ['@rushstack/eslint-config/profile/web-app', '@rushstack/eslint-config/mixins/react'],
+  parserOptions: { tsconfigRootDir: __dirname },
+  rules: {
+    'no-use-before-define': 'off',
+    '@typescript-eslint/no-use-before-define': 'off',
+    '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/no-floating-promises': 'off',
+    'react/jsx-no-bind': 'off'
```

Comment:
Why'd these get turned off?

Reply:
I fixed most of these but I left two:

`@typescript-eslint/no-floating-promises`
This one was making it quite difficult to use in my useEffect hooks, as essentially I was no longer able to use the following syntax:
```
useEffect(() => {
    async function loadLockfile(): Promise<void> {
    }
    loadLockfile(); <-- Throws error
  }, []);
```
And adding a `void loadLockfile()` causes a different eslint error, and I think this might be better than chaining on useless .then().catch() errors, do you know of a better solution?

`@rushstack/no-new-null`
I'm not sure what this rule is for, but it is blocking me from specifying a `null` return type on my react components, and prompts me to use `undefined` instead, but `undefined` is not a valid JSX element so it seems like this rule conflicts with standard react code?

Reply:
You can suppress those specific rules for those specific lines.

Reply:
changed